python/lsst/ts/MTMount/mtmount_csc.py

```diff
@@ -825,18 +865,19 @@ async def do_openMirrorCovers(self, data):
             commands.MirrorCoverLocksPower(on=False),
         )
 
-    async def do_disableCameraCableWrapTracking(self, data):
+    async def do_disableCameraCableWrapFollowing(self, data):
         self.assert_enabled()
-        self.camera_cable_wrap_task.cancel()
+        self.camera_cable_wrap_follow_start_task.cancel()
```

Comment:
Again, should you `await` for these cancelled tasks?

Reply:
I don't think it is necessary